recipe/meta.yaml

```diff
@@ -11,9 +11,10 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 1000
+  number: 1001
   script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
   skip: true  # [win]
+  include_recipe: False
```

Comment:
Why? In conda-forge the recipe is BSD and we encourage its distribution with the package.

Reply:
On my local conda build this causes the package to be about ~77 MB instead of ~11 MB

Reply:
<img width="977" alt="screen shot 2018-12-17 at 12 00 19 pm" src="https://user-images.githubusercontent.com/6001865/50112184-7568e480-01f3-11e9-8114-0d4482aa419e.png">
<img width="974" alt="screen shot 2018-12-17 at 12 00 36 pm" src="https://user-images.githubusercontent.com/6001865/50112190-7a2d9880-01f3-11e9-8dcc-dc4462df33c7.png">

Reply:
The only difference between build 0 and build 1 was adding `build.include_recipe: False`

Reply:
That is in your channel. Check the sizes on conda-forge:

https://anaconda.org/conda-forge/pyctest/files?version=0.0.10

You probably had the source code in the recipe directory there.

Reply:
Ah I see. Thanks